CMakeLists.txt

```diff
@@ -63,8 +63,36 @@ if(WIN32 AND (PROFILE OR VALGRIND_PROFILE OR GCOV OR STATIC_LINK_VW OR BUILD_JAV
   message(FATAL_ERROR "Unsupported option enabled on Windows build")
 endif()
 
+# This was the previous default but is only valid on x86_64 systems.
+set(linux_x86_64_opt_flags "")
+if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
+  set(linux_x86_64_opt_flags -msse2 -mfpmath=sse)
+endif()
+
+# Flags for optimal performance on arm64
+set(linux_arm64_opt_flags "")
+if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "aarch64|arm64|ARM64")
+
+  if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+    if("${CMAKE_CXX_COMPILER_VERSION}" VERSION_LESS "10.0.0")
+      set(linux_arm64_opt_flags -mcpu=cortex-a72)
```

Comment:
Do we need to handle these old versions of clang and gcc?

Reply:
After giving it a thought, no we don't. I will remove them!